OrbitGl/TimeGraph.cpp

```diff
@@ -214,6 +215,26 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
+//-----------------------------------------------------------------------------
+void TimeGraph::VerticalAlign(const TextBox* a_TextBox) {
```

Comment:
I have the feeling we need are better name for that function. Is `VerticallyScrollIntoView` appropriate or is this function doing more than that?

Reply:
Like it!